projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts

```diff
@@ -53,11 +59,23 @@ const moduleComponents = [
           component: OrderDetailShippingComponent,
         },
       },
+      features: {
+        consignmentTracking: '1.2',
+      },
     }),
+    SpinnerModule,
   ],
   providers: [OrderDetailsService],
-  declarations: [...moduleComponents],
+  declarations: [
+    ...moduleComponents,
+    TrackingEventsComponent,
+    ConsignmentTrackingComponent,
+  ],
   exports: [...moduleComponents],
-  entryComponents: [...moduleComponents],
+  entryComponents: [
+    ...moduleComponents,
+    TrackingEventsComponent,
+    ConsignmentTrackingComponent,
```

Comment:
It is better to move `TrackingEventsComponent`&`ConsignmentTrackingComponent` into `moduleComponents`.

Reply:
fixed